trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/VLM.py

```diff
@@ -334,4 +334,8 @@ def VLM(conditions,settings,geometry):
     
     alpha_i = np.arctan(Cdi_y/Cl_y) 
     
```

Comment:
I wonder since its only two things if it is worth creating a data structure. It may be confusing to other users why you didn't put all the other arguments in the data structure. The propeller outputs data structure on the other hand has a lot of outputs. You can just have something like " CL , CD , CM , _ _ _ = VLM() " for arguments that are not used.

Reply:
Mislabeled; I intended for it to be vortex_properties, but will change.